tests/helpers/test_many.ts

```diff
@@ -61,21 +48,24 @@ export const makeTestMany = <M extends string>(
         }
 
         const variation = matrix[name]()
-        ;(variation as TestRunnerOptions).getCurrentBundlerName = getCurrentBundlerName.bind(null, variation.config)
-
-        const rateLimitedTestFn = getRateLimitedTestFunction(testFn)
```

Comment:
Removed the rate limiting, which was a relict from ava. vitest does not run tests within a file in parallel so no need to rate limit.